docs/src/examples/requests/example.md

```diff
@@ -0,0 +1,115 @@
+## Examples
+### Sending requests
+
+This example shows how to send a requests to an http server
+
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {Test, expect, request, Response} from "vulcan/test.sol";
+
+contract RequestExample01 is Test {
+    function test() external {
+        Response memory getResponse = request.create().get("https://httpbin.org/get").send().unwrap();
+        Response memory postResponse = request.create().post("https://httpbin.org/post").send().unwrap();
+        Response memory patchResponse = request.create().patch("https://httpbin.org/patch").send().unwrap();
+        Response memory putResponse = request.create().put("https://httpbin.org/put").send().unwrap();
+        Response memory deleteResponse = request.create().del("https://httpbin.org/delete").send().unwrap();
+
+        expect(getResponse.status).toEqual(200);
+        expect(postResponse.status).toEqual(200);
+        expect(patchResponse.status).toEqual(200);
+        expect(putResponse.status).toEqual(200);
+        expect(deleteResponse.status).toEqual(200);
+    }
+}
+
+```
+
+### Sending a JSON payload
+
+This example shows how to send a request with a JSON body
+
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {Test, expect, request, Response, RequestClient, JsonObject} from "vulcan/test.sol";
+
+contract RequestExample02 is Test {
+    function test() external {
+        RequestClient memory client = request.create();
+
+        Response memory jsonRes = client.post("https://httpbin.org/post").json("{ \"foo\": \"bar\" }").send().unwrap();
```

Comment:
```suggestion
        Response memory jsonRes = client.post("https://httpbin.org/post").json('{ "foo": "bar" }').send().unwrap();
```

Reply:
This cannot be done. The contract parser fails when using single quotes for strings